bot/code_review_bot/revisions.py

```diff
@@ -269,17 +272,25 @@ def setup_try(self, tasks):
         def is_decision_task(task):
             image = task['task']['payload'].get('image')
             if image is not None and isinstance(image, str):
-                return image.startswith('taskcluster/decision')
+                return image.startswith('taskcluster/decision') or \
+                       image.startswith('djmitche/nss-decision')
```

Comment:
Nice image name :P
CC @djmitche

Reply:
Yeah, this was supposed to get rebuilt by someone with push permission, but I've forgotten who that was.